src/windows/2004/mono/amd64/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# escape=`

```

Comment:
what is this?

Reply:
No idea! I initially started with a Helix Server Core image, which used it. I assume it's responsible for changing the line break escape from backslash to backtick (given how problematic backslash is on Windows)

Let me check the docs.

https://docs.docker.com/engine/reference/builder/#escape

Yup!

Reply:
This is the relevant snippet from the doc @directhex references

> Setting the escape character to ` is especially useful on Windows, where \ is the directory path separator.

I have seen a number of people get stumped by Docker build failures as a result of the default escape character when building Windows Dockerfiles.  I recommend to everyone to change the escape character in Windows Dockerfiles.

Reply:
You have no idea what nightmares I had with AOT cross compiler builds on Windows caused by an escaped *space* in an msbuild exec call